mp-jwt/src/main/java/org/apache/tomee/microprofile/jwt/config/ConfigurableJWTAuthContextInfo.java

```diff
@@ -67,6 +67,8 @@
 
     private Config config;
     private JWTAuthContextInfo jwtAuthContextInfo;
+    private static final String PUBLIC_KEY_ERROR = "Could not read MicroProfile Public Key";
+    private static final String PUBLIC_KEY_ERROR_LOCATION = PUBLIC_KEY_ERROR + " from Location: ";
```

Comment:
I'm not sure if we should follow this pattern for message logging, considering that we don't do it for the entire codebase.

Reply:
I just didn't want the code to repeat the message a lot of times. What is the issue with this pattern?

Reply:
I think it is just an improvement on this class, but I think this is a good practice instead of replicating the same thing everywhere. If I find other places like this I will change.

Do you think this is a blocker for this PR? If not can you approve?

Reply:
No, not a blocker. Should be fine.